wandb/apis/importers/wandb.py

```
@@ -0,0 +1,1133 @@
+        skip_ids: Optional[List[str]] = None,
+        start_date: Optional[str] = None,
+    ):
+        config = ImportConfig(entity=dst_entity, project=dst_project)
+
+        reports = self.collect_reports(src_entity, src_project, reports_limit)
+        self.import_reports(reports, config, max_workers)
+
+        runs = self.collect_runs(
+            src_entity, src_project, runs_limit, skip_ids, start_date
+        )
+        self.import_runs(runs, config, max_workers)
+
+        sequences = list(
+            self.collect_artifact_sequences(
+                src_entity, src_project, artifact_sequences_limit
+            )
+        )
+        self.import_artifact_sequences(sequences, config, max_workers)
+        self.use_artifact_sequences(sequences, config, max_workers)
+
+        entity = src_entity if dst_entity is None else dst_entity
+        self._remove_placeholders(entity, dst_project)
+
+    def import_artifact_sequences(
+        self,
+        sequences: Iterable[ArtifactSequence],
+        config: Optional[ImportConfig] = None,
+        max_workers: Optional[int] = None,
+    ) -> None:
+        """Import a collection of artifact sequences.
+
+        Use `config` to specify alternate settings like where the report should be uploaded
+
+        Optional:
+        - `max_workers` -- set number of worker threads
+        """
+        with progress.live:
+            with ThreadPoolExecutor(max_workers) as exc:
+                futures = {
+                    exc.submit(self.import_artifact_sequence, seq, config): seq
+                    for seq in sequences
+                }
+                for future in progress.task_pbar.track(
+                    as_completed(futures),
+                    description="Artifact Sequences",
+                    total=len(futures),
+                ):
+                    try:
+                        future.result()
+                    except Exception:
+                        continue
```

Ignoring exceptions silently is not a good idea.